sources/packages/api/src/utilities/disbursements-utils.ts

```diff
@@ -17,6 +17,21 @@ export function getDisbursementValuesByType(
   return awards.filter((award) => types.includes(award.valueType));
 }
 
+/**
+ * Extract from the list of awards (disbursement values)
+ * the specific types.
+ * @param awards list of awards (disbursement values).
+ * @param valueCode valueCode to be extracted.
+ * @returns list of awards of the specified types.
+ */
+export function getDisbursementAmountByValueCode(
+  awards: Award[],
+  valueCode: string,
+): number {
+  return +awards.find((award) => award.valueCode.includes(valueCode))
+    .valueAmount;
```

Comment:
`.valueAmount;`??

Reply:
yes 😉

Reply:
what if the `.find()` returns undefined?. is that a possible scenario?